editor/src/components/editor/action-types.ts

```diff
@@ -998,6 +1000,11 @@ export interface SaveToGithub {
   targetRepository: GithubRepo
 }
 
+export interface SetFileBrowserDragState {
+  action: 'SET_DRAG_SESSION_STATE'
```

Comment:
Please can we also change this to reflect that it is specific to the file browser / image / asset dragging

Reply:
renamed it